### PR TITLE
blockdev_commit_throttle:commit to throttle node

### PR DESCRIPTION
**Pull request by aliang123**
Commit to throttle node

Signed-off-by: Aihua Liang <aliang@redhat.com>
id:2003941

### CODE REVIEW

**Comment by aliang123** [comment]
PASS 1-Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.blockdev_commit_throttle.q35

**Comment by aliang123** [comment]
@zhencliu @ngu-niny, please help to review, thanks.

File: qemu/tests/cfg/blockdev_commit_throttle.cfg [hunk 1]
```diff
@@ -0,0 +1,43 @@
+- blockdev_commit_throttle:
+    type = blockdev_commit_throttle
+    virt_test_type = qemu
+    only Linux
+    images += " data"
+    force_create_image_data = yes
+    force_remove_image_data = yes
```
Comment (zhencliu) [inline qemu/tests/cfg/blockdev_commit_throttle.cfg:7]: use remove_image_data instead

Comment (aliang123) [reply qemu/tests/cfg/blockdev_commit_throttle.cfg]: Have updated it, thanks.

**Comment by zhencliu** [comment]
LGTM

**Comment by ngu-niny** [comment]
PASS 1-Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.blockdev_commit_throttle

Acked-by: Nini Gu <ngu@redhat.com>

**Pull request merged**